lib/master/src/Prints.cc

```diff
@@ -8,23 +8,23 @@ namespace kickcat
     {
         std::stringstream os;
         os << "\n -*-*-*-*- slave " << std::to_string(slave.address) << " -*-*-*-*-\n";
-        os << "Vendor ID:       " << "0x" << std::setfill('0') << std::setw(8) << std::hex << slave.vendor_id << "\n";
-        os << "Product code:    " << "0x" << std::setfill('0') << std::setw(8) << std::hex << slave.product_code << "\n";
-        os << "Revision number: " << "0x" << std::setfill('0') << std::setw(8) << std::hex << slave.revision_number << "\n";
-        os << "Serial number:   " << "0x" << std::setfill('0') << std::setw(8) << std::hex << slave.serial_number << "\n";
+        os << "Vendor ID:       " << "0x" << std::setfill('0') << std::setw(8) << std::hex << slave.sii.vendor_id << "\n";
```

Comment:
Is it possible to use `std::endl` instead of `\n` for consistency sake?

Reply:
This PR has nothing to do with those prints, so I prefer to not add noise on this.